DeepLearningSuite/Tools/AutoEvaluator/autoEvaluator.cpp

```diff
@@ -124,7 +126,7 @@ class MyApp:public SampleGenerationApp{
 
                 GenericInferencerPtr inferencer(new GenericInferencer(inferencerConfig, inferencerWeights, inferencerNames, inferencerImplementation));
                 MassInferencer massInferencer(reader->getReader(),inferencer->getInferencer(), false);
-                massInferencer.process(useDepth, readerDetection);
+                massInferencer.process(useDepth, readerDetection, &time);
```

Comment:
Instead of passing any argument, just call massInferencer.inferencer->getMeanDurationnTime()